client/src/components/Carousel/Carousel.js

```diff
@@ -0,0 +1,217 @@
+`
+
+const CarouselSlideTitle = styled.h2`
+  position: relative;
+  margin: 0;
+  font-family: "Fira Sans", sans-serif;
+  font-size: calc(1em + (3vw - 0.5vmin));
+  font-weight: 700;
+  text-shadow: 2px 2px 8px rgba(0, 0, 0, 0.3);
+`
+
+const CarouselSlideDeveloper = styled.span`
+  margin-top: 0.5em;
+  color: #f9b2c8;
+  font-family: Vollkorn, Georgia, "Times New Roman", Times, serif;
+  font-size: calc(1em + (1vw - 0.5vmin));
+  font-style: italic;
+`
+const CarouselSlideDateAndPlatforms = styled.div`
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  margin-top: 0.5em;
+`
+
+const CarouselSlideDate = styled.span`
+margin-right: 1em;
+font-weight: 600;
+`
+
+const CarouselSlidePlatform = styled.span`
+margin-right: 0.5em;
+padding: 3px 12px;
+border-radius: 0.25em;
+background: #951010;
+font-size: 0.875em;
+font-weight: 600;
+`
+
+const CarouselSlideAddToSelectMenu = styled.select`
+width: min-content;
+height: 40px;
+margin-top: calc(1em + (1.3vw - 0.5vmin));
+padding: 10px 0 11px 15px;
+border: 0;
+border-radius: 4px;
+background-color: #fff;
+background-image: url("data:image/svg+xml;charset=US-ASCII,%3Csvg%20xmlns%3D%22http%3A%2F%2Fwww.w3.org%2F2000%2Fsvg%22%20width%3D%22292.4%22%20height%3D%22292.4%22%3E%3Cpath%20fill%3D%22%23FFFFFF%22%20d%3D%22M287%2069.4a17.6%2017.6%200%200%200-13-5.4H18.4c-5%200-9.3%201.8-12.9%205.4A17.6%2017.6%200%200%200%200%2082.2c0%205%201.8%209.3%205.4%2012.9l128%20127.9c3.6%203.6%207.8%205.4%2012.8%205.4s9.2-1.8%2012.8-5.4L287%2095c3.5-3.5%205.4-7.8%205.4-12.8%200-5-1.9-9.2-5.5-12.8z%22%2F%3E%3C%2Fsvg%3E"), linear-gradient(155.35deg, #1fee7e -28.1%, #2386bd 119.81%);
+background-repeat: no-repeat, repeat;
+background-position: right 1.3em top 50%, 0 0;
+background-size: .65em auto, 100%;
+color: #fff;
+font-family: "Open Sans", sans-serif;
+appearance: none;
+
+&:hover {
+  cursor: pointer;
+}
+`
+
+const CarouselSlideAddToSelectMenuOptions = styled.option`
+  padding-right: 1em;
+  color: #666;
+  font-size: 1.2em;
+`
+
+const CarouselSlideImage = styled.img`
+width: 100vw;
+height: 42vw;
+object-fit: cover;
+`
+
+const CarouselControls = styled.div``;
+
+const CarouselSlide = ({slide}) => {
+
+  const CarouselSlideListItem = styled.li`
+  position: absolute;
+  left: 0;
+  width: 100%;
+  max-width: 1440px;
+  height: 42vmax;
+  background: linear-gradient(90deg, rgba(0, 0, 0, 0.5) 0%, rgba(0, 0, 0, 0) 100%), url(${slide.image}) no-repeat;
+  background-position: center center;
+  background-size: cover;
+  box-shadow: inset 0 0 150px #000;
+`;
+
+  console.log(slide)
+  return (
+    <CarouselSlideListItem>
+      <CarouselSlideInfo>
+        <CarouselSlideTitle>
+          {slide.title}
+        </CarouselSlideTitle>
+        <CarouselSlideDeveloper>
+          {slide.developer}
+        </CarouselSlideDeveloper>
+        <CarouselSlideDateAndPlatforms>
+        <CarouselSlideDate>
+          {slide.releaseDate}
+        </CarouselSlideDate>
+        {slide.platforms.map(platform => {
```

Comment:
Instead of using the return keyword since you're only returning one component use an implict return:
`slide.platforms.map(platform => (
  <CarouselSlidePlatform title={platform.full}>{platform.short}</CarouselSlidePlatform>
        ))}
`